application.go

```diff
@@ -1,35 +1,31 @@
-package client
+package sdk
 
 import (
 	"context"
 
 	"github.com/cosmos/gogoproto/grpc"
 	"github.com/pokt-network/poktroll/x/application/types"
-
-	"github.com/pokt-network/shannon-sdk/sdk"
 )
 
-var _ sdk.ApplicationClient = (*applicationClient)(nil)
-
-// applicationClient is an ApplicationClient implementation that uses the gRPC
+// ApplicationClient is an ApplicationClient implementation that uses the gRPC
 // query client of the application module.
-type applicationClient struct {
-	queryClient types.QueryClient
+type ApplicationClient struct {
```

Comment:
Any reason for exporting `ApplicationClient` struct?

Reply:
Yes, the general pattern/best practice we need to follow, IMO, is to use public visibility for anything that can be reasonably accessed/used externally. There can be helper methods to make it easier to use the `ApplicationClient`, but making it public does not interfere with its operation in any way (similar to golang's standard library)

Reply:
Can you please start updating the docs to reflect this as well?

Even if it's just headers that we'll fill in later.

Reply:
Could you please clarify this? Do you mean opening a PR against the [sdk.md](https://github.com/pokt-network/poktroll/blob/e7800685a74babc4cc20624ac5474bf34b57abc8/docusaurus/docs/develop/packages/sdk.md) in the poktroll repo?
Or a docs directory in this repo?